docker-compose.yml

```diff
@@ -1,5 +1,27 @@
 version: "3.9"
 services:
+  coturn:
+    environment:
+      REALM: ret
+      PSQL: host=db dbname=ret_dev user=postgres password=postgres options='-c search_path=coturn' connect_timeout=30
+      TLS_PORT: 5349
+      MIN_PORT: 50000
+      MAX_PORT: 50050
+    build:
+      context: .
+      dockerfile: dockerfiles/coturn.Dockerfile
+    entrypoint: /bin/sh /entrypoint.sh
```

Comment:
What would you think about moving this line directly into the dockerfile?

Reply:
Oh yep, I see that you have been moving those. What the benefit of doing that? image reusability?

Reply:
It makes the compose file lighter and more focused on orchestration (rather than containerization). If that makes sense. 😄